docs/developers/plugins/payment/worldline.rst

```diff
@@ -0,0 +1,47 @@
+.. _developers_plugins_payment_worldline:
+
+========================
+Worldline payment plugin
+========================
+
+Developers can setup a local environment to test the Worldline payment plugin. See
+:ref:`configuration_payment_worldline` on how to setup a **Worldline merchant** and
+configure webhooks with the **Worldine Webhook configuration**.
+
+Testing webhooks with Ngrok
+===========================
+
+To locally test webhooks a service like https://ngrok.com can be used. To generate
+an **Auth Token** for local usage you will have to signup for an account. The following
+steps assume you have access to a local docker setup.
+
+1. After the registration process follow the  click on the **Your Authtoken** link
+   in the sidebar on the leftunder the **Getting Started** section.
+
+2. Click the **Copy** button to copy your **Authtoken** which will be used later on.
+
+3. Run the following command to start the Ngrok container which will redirect requests
+   from Worldline to your local setup:
+
+    .. code-block:: bash
+
+       docker run --net=host -it -e NGROK_AUTHTOKEN=<your-authtoken> ngrok/ngrok:latest http <django-server-listen-port>
```

Comment:
ah I just installed the (AUR) package I think, it's probably enough docs-wise to say something like "you can use ngrok to expose your local dev environment to the internet so that webhooks can be delivered" and it's up to the developer to get ngrok or a similar solution on their system :)